wgpu-native/src/pipeline.rs

```diff
@@ -209,6 +207,7 @@ pub struct PipelineStageDescriptor {
 pub struct ComputePipelineDescriptor {
     pub layout: PipelineLayoutId,
     pub stages: *const PipelineStageDescriptor,
+    pub stages_length: usize,
```

Comment:
The compute desc should only have a pointer to the compute stage, see https://github.com/gpuweb/gpuweb/blob/bce24a20f65f6344a1285bb03e7abc28a9d760ea/design/sketch.webidl#L560

Reply:
Gotcha. Fixed in subsequent commit